### PR TITLE
DAT-3720, add hyphens to infobox labels

### PR DESCRIPTION
**Pull request by RafLeszczynski**
## Links

* https://wikia-inc.atlassian.net/browse/DAT-3720
* https://github.com/Wikia/app/pull/9937/files

## Description

Add hyphens to infobox labels

## Reviewers

@SebastianMarzjan @rogatty @idradm

### CODE REVIEW

**Comment by SebastianMarzjan** [comment]
Looks good; I'm thinking about the lang property on the articles (we set it to userlang, not content-lang)

**Comment by rogatty** [comment]
@SebastianMarzjan are you sure about that?

In console:
```
> $('article').closest('[lang]')
< [<div class=​"page-wrapper row" lang=​"en" dir=​"ltr">​…​</div>​]
```

In the template that is rendered on server side:
```
<div class="page-wrapper" lang="{{wikiVariables.language.content}}" dir="{{wikiVariables.language.contentDir}}">
```

In the template that is used by Ember:
```
<div class="page-wrapper row" lang={{language.content}} dir={{language.contentDir}}>
```

File: front/main/app/styles/module/article/_portable-infobox.scss [hunk 1]
```diff
@@ -191,6 +191,7 @@ $hero-caption-line-height: 18px;
 
 		.pi-data-label {
 			flex-basis: 40%;
+			hyphens: auto;
```
Comment (vforge) [inline front/main/app/styles/module/article/_portable-infobox.scss]: Shouldn't we re-use this solution? https://github.com/Wikia/app/pull/9937

**Comment by SebastianMarzjan** [comment]
@rogatty Mental shortcut; I was referring to the MW app (Oasis skin) as part of this ticket. Good that we have that checked for Mercury too now, thanks!

File: front/main/app/styles/module/article/_portable-infobox.scss (hunk 1 above)
Comment (hakubo) [inline front/main/app/styles/module/article/_portable-infobox.scss]: in mercury we have autoprefixer - I think this should handle that so we don't have to write mixin for that.

**Comment by hakubo** [comment]
lgtm

**Pull request merged**